### ISSUE TITLE
Make initializedStoreOnce and initializedLanguageOnce props public

### ISSUE THREAD

**Issue by dobosalparbc** [i18next/i18next#2087]
## 🚀 Feature Proposal

Exposing initializedStoreOnce and initializedLanguageOnce props for i18n instance

## Motivation

useSSR hook of the react-i18next package doesn't update the language and the store of i18n instance if the language was changed on server side because the initializedStoreOnce and initializedLanguageOnce props are set to true.
Exposing them we would be able to reset those values on need.

## Example
This way we would be able to set the language server side with SSR. 

```
if (isOnServer() && i18n.language !== initialLanguageCode) {
        i18n.initializedLanguageOnce = false;
        i18n.initializedStoreOnce = false;
    }
useSSR(initialTranslationResource, initialLanguageCode);
```

**Comment by adrai** [comment]
Theoretically these are only for internal use... but feel free to provide a PR to address this

**Comment by dobosalparbc** [comment]
Looks like I don't have permission to push the branch

**Comment by adrai** [comment]
There's no need to push the branch... just fork the repository and create a PR: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by adrai** [comment]
addressed with v23.7.10
